### PR TITLE
Feat/data delivery to lims

### PR DESCRIPTION
**Pull request by patrikgrenfeldt**
## Description
*This PR adds/fixes data delivery to the data exported to LIMS*

### How to prepare for test
- [ ] ssh to relevant server (depending on type of change)
- [ ] Use stage: `us`
- [ ] paxa the environment: `paxa`
- [ ] install on stage (example for hasta):
`bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-cg-stage.sh [THIS-BRANCH-NAME]`

### How to test
- [ ] do submit order

### Expected test outcome
- [ ] check that data delivery was set on the lims sample
- [ ] Take a screenshot and attach or copy/paste the output.

## Review
- [ ] code approved by
- [ ] tests executed by
- [ ] "Merge and deploy" approved by
Thanks for filling in who performed the code review and the test!

### This [version](https://semver.org/) is a
- [ ] **MAJOR** - when you make incompatible API changes
- [x] **MINOR** - when you add functionality in a backwards compatible manner
- [ ] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

## Implementation Plan
- [ ] ~Document in ...~
- [x] create UDF Data Delivery in LIMS production
- [ ] Deploy this branch
- [ ] Inform to AG, data_analysis

### CODE REVIEW

**Comment by annagellerbring** [comment]
FYI @emmser

**Review by henrikstranneheim — APPROVED** [review]
👍

File: cg/constants/lims.py [hunk 1]
```diff
@@ -40,12 +41,6 @@
     "verified_organism": "Verified organism",
     "volume": "Volume (uL)",
     "well_position_rml": "RML well position",
-    "verified_organism": "Verified organism",
-    "sequencing_qc_pass": "Passed Sequencing QC",
-    "region": "Region",
-    "collection_date": "Collection Date",
-    "original_lab_address": "Original Lab Address",
-    "original_lab": "Original Lab",
```
Comment (Mropat) [inline cg/constants/lims.py:48]: this cannot be deleted

**Review by Mropat — CHANGES_REQUESTED** [review]
Approved to add new UDF, but dont delete the old ones that are needed.
Where is the code for parsing data delivery into lims?

**Pull request merged**